reference/tools/system/package_manager.rst

```diff
@@ -0,0 +1,273 @@
+* *Chocolatey* for **Windows**
+* *Zypper* for **Linux** with distribution names: *opensuse*, *sles*
+* *Pkg* for **Linux** with distribution names: *freebsd*
+* *PkgUtil* for **Solaris**
+
+You can override this default mapping and set the package manager tool you want to use by
+default setting the configuration property `tools.system.package_manager:tool`.
+
+.. _conan_tools_system_package_manager_methods:
+
+Methods available for system package manager tools 
+--------------------------------------------------
+
+All these wrappers share three methods that represent the most common operations with a
+system package manager. They take the same form for all of the package managers except for
+*Apt* that also accepts the *recommends* argument for the :ref:`install
+method<conan_tools_system_package_manager_apt>`.
+
+* ``install(self, packages, update=False, check=False):`` try to install
+  the list of packages passed as a parameter. If the parameter ``check`` is ``True`` it
+  will check if those packages are already installed before installing them. If the
+  parameter ``update`` is ``True`` it will try to update the package manager database
+  before checking and installing. Its behaviour is affected by the value of
+  ``tools.system.package_manager:mode``
+  :ref:`configuration<conan_tools_system_package_manager_config>`.
+* ``install_substitutes(packages_substitutes, update=False, check=True)``: try to install
+  the list of lists of substitutes packages passed as a parameter, e.g., ``[["pkg1", "pkg2"], ["pkg3"]]``.
+  It succeeds if one of the substitutes list is completely installed, so it's intended to be used when you have
+  different packages for different distros. Internally, it's calling the previous
+  ``install(packages, update=update, check=check)`` method, so ``update`` and ``check`` have the same
+  purpose as above.
+* ``update()`` update the system package manager database. Its behaviour is affected by
+  the value of ``tools.system.package_manager:mode``
+  :ref:`configuration<conan_tools_system_package_manager_config>`.
+* ``check(packages)`` check if the list of packages passed as parameter are already
+  installed.
+
+.. _conan_tools_system_package_manager_config:
+
+Configuration properties that affect how system package managers are invoked
+----------------------------------------------------------------------------
+
+As explained above there are several ``[conf]`` that
+affect how these tools are invoked:
+
+* ``tools.system.package_manager:tool``: to choose which package manager tool you want to
+  use by default: ``"apt-get"``, ``"yum"``, ``"dnf"``, ``"brew"``, ``"pacman"``,
+  ``"choco"``, ``"zypper"``, ``"pkg"`` or ``"pkgutil"``
+
+* ``tools.system.package_manager:mode``: mode to use when invoking the package manager
+  tool. There are two possible values:
+
+  * ``"check"``: will not try to update the package manager database or install any
+    packages in any case. This is the default value.
```

Comment:
I think this is explained too late to be the default behavior. I would explain the default behavior mostly at the beginning linking this.